apps/web/tests/dashboards/create-dashboards/entire-workspace-dashboard.spec.ts

```diff
@@ -34,14 +34,20 @@ test.describe('Entire Workspace Dashboard', () => {
         });
 
         await test.step('5. Check the created dashboard', async () => {
-            const locatorScope = page.locator('.p-breadcrumbs > span:nth-child(2) .link');
-            const locatorName = page.locator('.lnb-menu-item .selected .text');
+            const locatorScope = await page.locator('.p-breadcrumbs > span:nth-child(2) .link');
+            const locatorName = await page.locator('.lnb-menu-item .selected .text');
 
             await expect(locatorScope).toContainText(' Workspace ');
             await expect(locatorName).toContainText(dashboardName);
 
             // Todo: to be added
             // await expect(locatorName).toContainText('public');
         });
+
+        await test.step('6. Delete dashboard', async () => {
+            const deleteButton = await page.locator('.dashboard-title-icon-buttons-wrapper .delete-button');
+            await deleteButton.click();
+            await page.getByRole('button', { name: 'Confirm' }).click();
+        });
```

Comment:
Suggestion: It seems better to validate each test in a single test step using 'expect'!

Reply:
Thank you for suggestion! I'll update that!